README.md

```diff
@@ -183,6 +183,7 @@ Provisioning.PasswordCryptSaltLength=10
 ResourceDisk.Format=y
 ResourceDisk.Filesystem=ext4
 ResourceDisk.MountPoint=/mnt/resource
+ResourceDisk.MountOptions=None
```

Comment:
I think it should be `if mount_options is not None and mount_options != "None"`. You should also have a test case for this, i.e. `"None"` rather than `None`